README.md

```diff
@@ -101,7 +99,6 @@ npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standa
 
 ## 👥 Authors <a name="Ahmed"></a>
 
-> Mention all of the collaborators of this project.
 
 👤 **Ahmed**
```

Comment:
- [ ] I would kindly suggest you to update your readme by includes all your social media handle your twitter ,githube and linkedin as this is a means for a recruilter and developers to contact you